packages/prices/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@js-ligo/prices",
+  "version": "1.0.0",
+  "description": "",
+  "type": "module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build:clean": "del dist",
+    "build:js": "swc src -d ./dist --config-file ../../.swcrc",
+    "build:types": "tsc --emitDeclarationOnly --skipLibCheck",
+    "build": "pnpm build:clean && pnpm build:types && pnpm build:js",
+    "test:it": "node --experimental-vm-modules ../../node_modules/jest/bin/jest.js",
```

Comment:
This can be changed to `test` instead of `test:it` since it is just normal unit tests.